worker/raft/raftleaseconsumer/handler.go

```diff
@@ -0,0 +1,178 @@
+// Copyright 2021 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+package raftleaseconsumer
```

Comment:
I think I'd call this `leaseservice` and rename everything accordingly.

_Edit: I see the "raft" prefix convention with all the other packages now (sigh). Maybe "raftleaseservice"?_